config/default.js

```diff
@@ -42,11 +42,13 @@ module.exports = {
     sha: '',
     ref: '',
   },
-  elife: {
-    isPublic: true,
+  server: {
     api: {
       url: 'https://api.elifesciences.org/',
```

Comment:
this goes [through the CDN](https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L431) so to get a similar experience to the other environments you should point to [https://prod--gateway.elifesciences.org](https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L379)